packages/subgraph/manifest/data/arbitrum.json

```diff
@@ -10,9 +10,9 @@
       }
     ],
     "Plugin": {
-      "name": "MyPlugin",
-      "address": null,
-      "startBlock": 7727664
+      "name": "AdminRepo",
```

Comment:
I named it differently: https://github.com/aragon/multisig-plugin/pull/4/commits/86a9b38d3ba177e54870ed38c3ceaad104958c42

What's the intended naming?

Reply:
[We agreed on changing the struct to be more clear](https://aragonassociation.atlassian.net/browse/OS-1148)
It will now be 
```
{
  "info": "# Do not edit subgraph.yaml,this is a generated file. \n# Instead, edit subgraph.placeholder.yaml and run: yarn manifest",
  "network": "arbitrum-one",
  "PluginRepo": {
      "ensDomain": "my.plugin.dao.eth",
      "address": null,
    }
  "dataSources": {
    "PluginSetupProcessors": [
      {
        "name": "PluginSetupProcessor",
        "address": "0x308a1DC5020c4B5d992F5543a7236c465997fecB",
        "startBlock": 145462184
      }
      ]
  }
}
``` 

so will need to set the ens name